src/index.js

```diff
@@ -1,21 +1,60 @@
+import checkTask from './module/task-survey.js';
+import descriptionOnFocus from './module/focus-functionality.js';
+// eslint-disable-next-line import/no-cycle
+import {
+  addNewItem, clearAll, updateInput, removeItem,
+} from './module/add-remove-implementation.js';
+
 import './style.css';
 
-const toDoItems = [
-  { description: 'Do the chores', completed: false, index: 0 },
-  { description: 'Take a nap', completed: false, index: 1 },
-  { description: 'Do Microverse Work', completed: false, index: 2 },
-];
+// SET TODO ARR
+let toDoArr = [];
+export const setToDoArr = () => {
+  if (JSON.parse(localStorage.getItem('myToDos'))) {
+    toDoArr = JSON.parse(localStorage.getItem('myToDos'));
+  } else {
+    toDoArr = [];
+  }
+};
+setToDoArr();
 const listWrap = document.querySelector('.todo-list-wrap');
 // IMPLEMENT TO DO ITEMS TO SCREEN
-const implementToDoItems = () => {
-  toDoItems.forEach((task) => {
-    listWrap.innerHTML += `<li class="list-item" data-key="${task.index}">
+export const implementToDoItems = (arr) => {
+  arr.forEach((item) => {
+    let isCompleted = '';
+    let completedClass = '';
+    if (item.completed) {
+      isCompleted = 'checked';
+      completedClass = 'completed-item';
+    }
+    listWrap.innerHTML += `<li class="list-item ${completedClass}" data-key="${item.index}" draggable="true">
       <div class="list-item-check-name">
-        <button type="button" class="check-item" data-key="${task.index}"></button>
-        <input class="description-input" type="text" value="${task.description}">
+         <input type="checkbox" class="check-item" data-key="${item.index}" ${isCompleted}>
+  <input class="description-input" data-key="${item.index}" type="text" value="${item.description}">
       </div>
-      <button data-key="${task.index}" class="move-item"><i class="fas fa-ellipsis-v"></i></button>
+      <button type="button" data-key="${item.index}" class="move-item"><i class="fas fa-ellipsis-v"></i></button>
+      <button type="button" data-key="${item.index}" class="remove-item"><i class="far fa-trash-alt"></i></button>
     </li>`;
```

Comment:
- [ ] Kindly when I try to reload the page the tasks get doubled, you will find that you have more tasks than you actually have on the local storage, this issue is because when the page load you have to remove the displayed tasks and display them from the beginning because when you display the tasks again without removing the existing ones on the page you will end with display the same tasks again.

![image](https://user-images.githubusercontent.com/35535130/161124653-f1850284-0ccd-4747-a1d9-7303287eca9a.png)